demo/scss/index.scss

```diff
@@ -124,7 +154,18 @@ dialog::backdrop {
   animation-timing-function: var(--ease-elastic-in-3);
 }
 
-.slide-fade {
+.spin, .spin-back {
+  animation: var(--animation-spin) forwards;
+  animation-duration: 0.5s;
+  animation-timing-function: var(--ease-out-3);
```

Comment:
Is it possible to use a single line to configure the animation?

Reply:
I've done it. I've also moved these animations in their own file. Thanks for the feedback !